server/courselist.rb

```diff
@@ -46,9 +47,23 @@ class CourseList < Sinatra::Base
   # where needed.
   set :latte_config, config_hash
 
-  # Location to find the configuration files.  This can't be overridden by values
-  # in the configuration file because this is what identifies the location
-  # of the file to read.
+  # print environment when debugging level.
+  ENV.each_pair do |key,value|
```

Comment:
Ok.  I just figured out how to do that.

On Thu, Jul 9, 2015 at 1:01 PM, Lance E Sloan <notifications@github.com>
wrote:

> In server/courselist.rb
> <https://github.com/tl-its-umich-edu/StudentDashboard/pull/165#discussion_r34277387>
> :
>
> > @@ -46,9 +47,23 @@ class CourseList < Sinatra::Base
> >    # where needed.
> >    set :latte_config, config_hash
> >
> > -  # Location to find the configuration files.  This can't be overridden by values
> > -  # in the configuration file because this is what identifies the location
> > -  # of the file to read.
> > +  # print environment when debugging level.
> > +  ENV.each_pair do |key,value|
>
> This loop will execute whether or not debug logging is enabled. It will
> access each env. variable and it may or may not log anything. This is an
> unnecessary use of processing time. You need to wrap the loop with a check
> for debug mode instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/StudentDashboard/pull/165/files#r34277387>
> .
>